quarkus-extension/engine/runtime/src/main/java/org/camunda/bpm/quarkus/engine/extension/QuarkusProcessEngineConfiguration.java

```diff
@@ -44,6 +45,8 @@ public QuarkusProcessEngineConfiguration() {
     setTransactionsExternallyManaged(true);
 
     setIdGenerator(new StrongUuidGenerator());
+
+    setHistoryLevel(HistoryLevel.HISTORY_LEVEL_FULL); // Cockpit needs it
```

Comment:
We should use the `history` property. The `historyLevel` property will override whatever is set on the `history` property, which might look like incorrect behavior to users that only set the `history` property.

```suggestion
    setHistory(HISTORY_FULL); // Cockpit needs it
```